pkgs/foundryvtt/default.nix

```diff
@@ -166,12 +166,12 @@ let
 
         ln -s "$foundryvtt/public" "$out/public"
 
-        # Run PNG images through `pngout` if it’s available.
+        # Run PNG images through `oxipng` if it's enabled.
+        # could add "-o max" and "--zopfli" to increase compression power
+        # --interlace 0 or 1 to dis-/enable interlacing
         ${
-          if usePngout then
-            ''
-              find $foundryvtt/public -name '*.png' -exec ${pngout}/bin/pngout {} -k1 -y \;
-            ''
+          if useOxipng then
+            "${oxipng}/bin/oxipng $foundryvtt/public/**/*.png -s"
```

Comment:
`pngout -k1` keeps all chunks, but `oxipng -s` strips some of them. Have you noticed any visual issues when using `-s`?

(IIRC I ran into issues with stripping some chunks in the past and defaulted to using `-k1` after that, though anymore I usually convert PNGs to lossless JPEG XL or WebP depending on the intended use.)